app/Major.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Relations\BelongsToMany;
+
+class Major extends Model
+{
```

Comment:
The soft deletes trait is missing

Reply:
I don't think it would be necessary to delete majors ever, soft or otherwise. The policy is also written to disallow it.

Do you still want to enable soft deletes?

Reply:
No, it's fine without that. I thought the migration had added the column but I see that it hadn't. The `$guarded` array still contains `deleted_at` so I assumed it was meant to be there.

Reply:
Oops, yeah, that's a bad copy paste.